### PR TITLE
fixes docker and updates node

### PR DESCRIPTION
**Pull request by aliuk2012**

### CODE REVIEW

**Comment by aliuk2012** [comment]
please test this

File: cla_frontend/settings/docker.py [hunk 1]
```diff
@@ -8,7 +8,7 @@
     SECRET_KEY = "CHANGE_ME"
     pass
 
-DEBUG = True if os.environ.get('SET_DEBUG') == 'True' else False
+DEBUG = True if os.environ.get('DEBUG') == 'True' else False
```
Comment (ibrechin) [inline cla_frontend/settings/docker.py]: This should probably be
```
DEBUG = os.environ.get('DEBUG', 'False') == 'True'
````
to make sure it doesn't error if the env variable is not set.

Comment (aliuk2012) [reply cla_frontend/settings/docker.py]: 👍  Good point

**Comment by ibrechin** [comment]
retest this please

**Comment by ibrechin** [comment]
retest this please

**Pull request merged**